vyxal/elements.py

```diff
@@ -6115,6 +6115,19 @@ def zero_slice(lhs, rhs, ctx):
     (num, any) -> b[0:a]
     (str, str) -> regex.findall(pattern=a,string=b) (Find all matches for a regex)
     """
+    if type(lhs) == types.FunctionType:
+        return zero_slice(rhs, lhs, ctx)
+    if type(rhs) == types.FunctionType:
+        i = iterable(lhs, ctx=ctx)
```

Comment:
I don't wanna be that guy, but could you name this something more descriptive? Not sure what `i` stands for

Reply:
`i` stands for `iterable` lol

Reply:
Oh ok